esx_service/cli/vmdkops_admin.py

```diff
@@ -949,7 +949,7 @@ def tenant_create(args):
     """ Handle tenant create command """
     error_info, tenant = auth_api._tenant_create(name=args.name,
                                                  default_datastore=args.default_datastore,
-                                                 description="",
+                                                 description=args.description,
```

Comment:
I don't think the CLI parser fills None for arguments not provided. Description is an optional argument and looks like its set to "".  Below _description_ isn't provided and tenant_update in the CLI code uses args.description as is. 

/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup update --default-datastore sharedVmfs-1 --name grp1
vmgroup modify succeeded

 /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup ls
Uuid                                  Name  Description  Default_datastore  VM_list
------------------------------------  ----  -----------  -----------------  -------
35779376-dd7c-4d6c-973c-e18607225541  grp1               sharedVmfs-1